modules/experimental-layers/src/gpu-screen-grid-layer/gpu-screen-grid-layer.js

```diff
@@ -59,12 +59,12 @@ export default class GPUScreenGridLayer extends Layer {
     /* eslint-disable max-len */
     attributeManager.addInstanced({
       instancePositions: {size: 3, update: this.calculateInstancePositions},
-      // TODO: Needed, since attributeManger.update() only updates existing attributes.
       instanceCounts: {
         size: 4,
         transition: true,
         accessor: ['getPosition', 'getWeight'],
-        update: this.calculateInstanceCounts
+        update: this.calculateInstanceCounts,
+        isExternalBuffer: true
```

Comment:
`isExternalBuffer` was designed to be an internal state to `LayerAttribute`. The intention was to allow the user to override the attribute values via layer props. This feels like a hack.